ansible/inventories/production/group_vars/ungrouped-catalog-next/vars.yml

```
@@ -0,0 +1,5 @@
+---
+# TrendMicro
+trendmicro_policy_id_web: "{{ vault_trendmicro_policy_id_web }}"
```

I think it's more like this. You want to set the policy_id to the web_id.

```suggestion
trendmicro_policy_id: "{{ trendmicro_policy_id_web }}"
```